index.html

```diff
@@ -92,89 +92,9 @@ <h2 id="works-title">My recent works</h2>
-          when an unknown printer took a standard dummy text.</p>
-        <ul class="flex-row">
-          <li>css</li>
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
-      </div>
+    <article id="multipost"> <!-- class="work-popup" -->
     </article>
-    <div id="works-grid">
-      <article class="flex-column">
-        <h3>Professional Art Printing Data</h3>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups
-          required. has been the industry's standard</p>
-        <ul class="flex-row">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
-      </article>
-      <article class="flex-column">
-        <h3>Professional Art Printing Data</h3>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups
-          required. has been the industry's standard</p>
-        <ul class="flex-row">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
-      </article>
-      <article class="flex-column">
-        <h3>Professional Art Printing Data</h3>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups
-          required. has been the industry's standard</p>
-        <ul class="flex-row">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
-      </article>
-      <article class="flex-column">
-        <h3>Professional Art Printing Data</h3>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups
-          required. has been the industry's standard</p>
-        <ul class="flex-row">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
-      </article>
-      <article class="flex-column">
-        <h3>Professional Art Printing Data</h3>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups
-          required. has been the industry's standard</p>
-        <ul class="flex-row">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
-      </article>
-      <article class="flex-column">
-        <h3>Professional Art Printing Data</h3>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups
-          required. has been the industry's standard</p>
-        <ul class="flex-row">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
-      </article>
+    <div id="works-grid">  
```

Comment:
- [ ] Kindly the responsive design can improve, just add the buttons inside your content and try to add padding/margin in the y-axis for your text which describes your project.

![image](https://user-images.githubusercontent.com/60164239/131872115-4c97cf28-c082-4555-b3b4-bbbe0aba8f84.png)

![image](https://user-images.githubusercontent.com/60164239/131872462-4c375f49-a574-44ac-96d1-bbbb1b903ddb.png)